public/media_engagement/worldbank_plr_sms_traffic.js

```diff
@@ -45,10 +45,18 @@ firebase.auth().onAuthStateChanged(function (user) {
 
 const TIMEFRAME_WEEK = 7;
 const TIMEFRAME_MONTH = 30;
-var chartTimeUnit = "1day";
+const EXTEND_X_AXIS_BY_HOURS = 32
+var chartTimeUnit = "10min";
```

Comment:
I didn't intend to make the change on this PR  but since it was approved on imaqal, I thought it was better to leave it there than to uncommit the changes.